.env.sample

```diff
@@ -0,0 +1,3 @@
+# shellcheck shell=bash
+export GOMAXPROCS=6
```

Comment:
should we be setting GOMAXPROCS since it's highly dependent on the dev environment?

Reply:
This is a sample file, it's just to give some "idea" for people for their `.env`. It's not sourced anywhere.